c2rust-ast-exporter/build.rs

```diff
@@ -181,7 +184,14 @@ fn build_native(llvm_info: &LLVMInfo) {
             "clangRewrite",
             "clangLex",
             "clangBasic",
-        ] {
+        ];
+        if llvm_info.llvm_major_version >= 15 {
+            // insert after clangSema
```

Comment:
Is it order sensitive?

Reply:
Generally, library sequences do matter to linkers, even though modern linkers sometimes tolerate arbitrary sequences. It might not in this special case, but would take some effort to find out, and more to be sure it stays that way. https://www.linuxtopia.org/online_books/an_introduction_to_gcc/gccintro_18.html has a good summary

Reply:
Thanks for weighing in @chrysn! I thought I had provided an answer earlier (that order indeed matters since linkers resolve symbols in libraries from left to right) but I must have messed up somehow since I don't see my reply.